lib/paperclip/storage/ftp.rb

```diff
@@ -109,8 +110,19 @@ def with_ftp_servers(&blk)
         end
       end
 
+      def find_servers(servers)
+        if servers.respond_to?(:call)
+          servers.call(self)
+        elsif servers.is_a? Array
```

Comment:
I don't think we should check for the `Array` type here, as previously any object that implemented `each` was good enough. 

So the method could be simplified to:
```ruby
if servers.respond_to?(:call)
  servers.call(self)
else
  servers
end
```
or even
```ruby
servers.respond_to?(:call) ? servers.call(self) : servers
```